setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name="simpleocr",
+    packages=["simpleocr"],
+    version="1.0.0",
+    description="A library for simple OCR in Python using OpenCV",
+    author="The simple-ocr-opencv authors",
+    url="https://www.github.com/goncalopp/simple-ocr-opencv",
+    download_url="https://www.github.com/goncalopp/simple-ocr-opencv/archive/1.0.0.tar.gz",
+    keywords=["OCR", "OpenCV"],
+    license="AGPL",
+    classifiers=["Programming Language :: Python :: 2.7",
+                 "Programming Language :: Python :: 3",
+                 "License :: OSI Approved :: GNU Affero General Public License v2 or later (AGPLv2+)"],
+    include_package_data=True,
+    install_requires=["opencv-python"]
```

Comment:
@goncalopp This is what is breaking the Travis-CI for this branch. I could turn this into the following line:
```python
install_requires=[] if "TRAVIS" in os.environ else ["opencv-python"]
``` 

including an 

```python 
import os
```
at the top of the file, but this would install `OpenCV 3.2` (currently) by default for all platforms through `pip`. I have tested on Ubuntu 17.04, and it installs `opencv-python` with the `manylinux` wheel just fine for both Python 2 and 3. Removing this header will mean that users have to figure out how to install `opencv-python` on their own, both on Windows and Linux. It could also be made so only Windows platforms install `opencv-python` automatically through `pip`, and Linux just gets a `None` value. What is your opinion on this?